### DISCUSSION TITLE
VirusTotal (Grindsoft actually) still flag's cypress as a trojan. It has been months already.

### DISCUSSION THREAD

**Discussion by mauriciocarnieletto** [cypress-io/cypress#25523] [Questions and Help] [answered]
Hey guys!
I've been trying to adopt Cypress as the main test framework here at next bank, and our security team keeps complaining about this report

https://www.virustotal.com/gui/file/e4b3570c13993e1868bea18982336c0e867df72a4d2777dd9d742b0f6f44deea/detection

I'd already sent an email to GrindSoft, asking them to reconsider it (cause of course it's bu*t), tho, they don't even get to respond.

Can anything be done about that? I really, really want to make Cypress our official testing tool.

Does anyone else have suffered from that?

This is related to https://github.com/cypress-io/cypress/issues/6942

**Comment by nagash77** [comment, marked as answer]
@mauriciocarnieletto thank you again for your support. I have escalated this internally and our security team will be reaching out to VirusTotal (and perhaps others) to make sure they remove Cypress as a perceived threat.

**Comment by cypressmacaire** [comment]
Hello we have reached out to the vendor of the scanner on multiple occasions.  Eventually they were responsive, but have not removed the finding.  Virustotal acknowledges they don't control these third party engines and that even 1 or 5 out of 60 engines does not mean it is not a false positive. Gridinsoft is not a notable provider of security software for discovering malware. The detection was based on keywords in what is know as a yara detections file, and based on the behavior of the application.  If a word contained in this file was anywhere in the app it would fire off the detection. Our software undergoes continuous security testing and we do listen closely to the community and our users.  We will continue to get this false positive removed, but we care confident that our software is safe and will do our best to assure everyone using cypress to make their work better.
